src/table/tableview/table-view.component.js

```diff
@@ -41,6 +42,9 @@ angular.module('patternfly.table').component('pfTableView', {
         $log.debug("$onInit");
       }
 
+      if (angular.isDefined(ctrl.colummns)) {
+        ctrl.columns = ctrl.colummns;
+      }
```

Comment:
should probably check if columns is undefined. If both are set (for some strange reason, we should use columns over columms)